xclim/testing/tests/test_indices.py

```diff
@@ -114,7 +114,7 @@ def test_simple(self, tasmin_series):
 
         out = xci.cold_spell_duration_index(tn, tn10, freq="YS")
         assert out[0] == 10
-        assert out.units == "days"
+        convert_units_to(out, "days")
```

Comment:
Why remove the explicit check ?

Reply:
Because the function returns "d" as units, because of the way it is converted from a pint Quantity with `f"{q:~}"`. The specific string of the unit could change if we decide "days" is the default str? I just wanted this to be a bit more flexible.

Reply:
but this will work even if the units is hour